Public/Src/Utilities/Configuration/Mutable/ScheduleConfiguration.cs

```diff
@@ -72,6 +72,7 @@ public ScheduleConfiguration()
 
             UnsafeDisableSharedOpaqueEmptyDirectoryScrubbing = false;
             InputChanges = AbsolutePath.Invalid;
+            // PinBeforePlace = false;

```

Comment:
Remove or uncomment?

Reply:
Added a comment on the first iteration.

---
In reply to: [327867667](https://github.com/microsoft/BuildXL/pull/925#discussion_r327867667) [](ancestors = 327867667)